Common/include/option_structure.hpp

```diff
@@ -2237,6 +2242,41 @@ static const MapType<std::string, POD_KIND> POD_Map = {
   MakePair("STATIC_POD",      POD_KIND::STATIC)
   MakePair("INCREMENTAL_POD", POD_KIND::INCREMENTAL)
 };
+enum ENUM_BODY_FORCE_TYPE {
+    CONSTANT_BF = 0,			/*!< \brief Constant body force over domain using vector. */
+    VARIABLE_BF = 1,				/*!< \brief Body force model that is spatially varying. */
```

Comment:
enum class and do not add numbers explicitly, we do not want to rely on them.
```suggestion
enum BODY_FORCE_TYPE {
    CONSTANT,			/*!< \brief Constant body force over domain using vector. */
    VARIABLE,				/*!< \brief Body force model that is spatially varying. */
```
replace tabs by 2 spaces in the code and fix the indentation please, in your new files the easiest thing is to run clang-format, the SU2 format style file is in the root.